modules/dkan/dkan_fixtures/modules/dkan_default_content/data/dkan_data_story_show%3Fid=e27d899b-6d72-49c9-8807-b49424c4cf70.json

```diff
@@ -25,7 +25,7 @@
                 "id": "80a9cc2b-c817-405e-9591-14565759defc"
             }
         ],
-        "field_image": "files/dkan_data_story/A_nurse_vaccinates_Barack_Obama_against_H1N1_0.jpg",
+        "field_image": "files/dkan_data_story/immunization-of-children-in-doctors-office-725x479.jpg",
         "field_image_alt": "Nurse vaccinates Pres. Obama",
```

Comment:
Need to remove "obama" from the alt description

Reply:
oops.